operators/nftopology/config/default/kustomization.yaml

```diff
@@ -0,0 +1,72 @@
+# Adds namespace to all resources.
+namespace: nftopology-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: nftopology-
```

Comment:
I think this works to fix the manifests, but does it fix the Makefile variable? I am not sure